src/services/cohortService.ts

```diff
@@ -40,11 +40,23 @@ export const getCohortsService = async (searchString: string) => {
   return await getCohortsQuery(searchString)
 }
 
+export const generateCohortIdService = async () => {
+  let cohortId = 1
+  const lastCohort = await Cohort.findOne().sort({ cohortId: -1 })
+  if (lastCohort?.cohortId) {
+    cohortId = parseInt(lastCohort.cohortId, 10) + 1
+  }
+
+  return String(cohortId).padStart(6, "0")
```

Comment:
I think returning something like 1 instead of 000001 is much better for the user experience.
```suggestion

  return cohortId
```